flexget/components/parsing/plugin_parsing.py

```diff
@@ -38,6 +38,8 @@ def init_parsers(manager):
 class PluginParsing:
     """Provides parsing framework"""
 
+    _selected = False
```

Comment:
What does this do? Why not just check if `selected_parsers` is empty, rather than keeping track separately? Otherwise it looks good to me.

Reply:
I do this check because to avoid pop a entry on a task that didn't set it! Otherwise a inner task that does not set a config would delete the config from the upper one

Reply:
Ahh, understood. I'm not sure making it a class property helps though does it? If another task runs before the exit handler of the current task, this will still be True there, even though that task didn't set it, no?

Reply:
Maybe something like `task._custom_parser = True`, then do `getattr(task, '_custom_parser', False)` to check.